packages/maskbook/src/plugins/RedPacket/types.ts

```diff
@@ -36,6 +36,11 @@ export enum DialogTabs {
     past = 1,
 }
 
+export enum RpTypeTabs {
```

Comment:
This should be a local enum in packages/maskbook/src/plugins/RedPacket/SNSAdaptor/RedPacketCreateNew.tsx